### PR TITLE
Fix docs build failed while building pdf file

### LINKED ISSUE #695: NumPyro master and PyPI versions not in sync
Hi - I was running some examples that use newly released features on the master branch and noticed that the PyPI version seems to be based on the version 0.2.4 from January 2020.  I wonder if this is because the minor versions need to be incremented -- many new features have been added since January 2020, but the minor version has not been incremented.  Not sure how this package is updated on PyPI when there are updates to master.  Just wanted to surface this.

### PR DESCRIPTION
**Pull request by fehiepsi**
I think this and the previous PR will also resolve #695. examples/tutorials are synced with the master branch in the `latest` readthedocs version.

Currently, our [readthedocs build](https://readthedocs.org/projects/numpyro/builds/) is failing because it cannot generate pdf file from the source. This PR fixes those latex issues. This also fixes [Edit to Github](http://num.pyro.ai/en/latest/tutorials/bayesian_regression.html) button on the right top, which pointed to a not-available location (e.g.  `docs/source/tutorials/bayesian...` file, which is deleted in the source and not commited).

### CODE REVIEW

File: docs/source/_templates/breadcrumbs.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{%- extends "sphinx_rtd_theme/breadcrumbs.html" %}
```
Comment (fehiepsi) [inline docs/source/_templates/breadcrumbs.html:1]: We extend [this file](https://github.com/readthedocs/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/breadcrumbs.html) to modify `Edit on GitHub` link for examples.

File: docs/source/conf.py [hunk 2]
```diff
@@ -266,7 +271,10 @@
 
     # Additional stuff for the LaTeX preamble.
     #
-    # 'preamble': '',
+    'preamble': r'''
+        \usepackage{pmboxdraw}
```
Comment (fehiepsi) [inline docs/source/conf.py:275]: this is needed to render tqdm progressbar characters

File: docs/source/conf.py [hunk 3]
```diff
@@ -266,7 +271,10 @@
...
+        \usepackage{alphabeta}
```
Comment (fehiepsi) [inline docs/source/conf.py:276]: This is needed for Greece character \alpha \gamma in some tutorial.

File: notebooks/source/bayesian_hierarchical_linear_regression.ipynb [hunk 4]
```diff
@@ -216,7 +216,6 @@
     "\\sigma &\\sim \\mathcal{N}(0, 100) \\\\\n",
     "FVC_{ij} &\\sim \\mathcal{N}(\\alpha_i + t \\beta_i, \\sigma)\n",
     "\\end{align}\n",
-    "$$\n",
```
Comment (fehiepsi) [inline notebooks/source/bayesian_hierarchical_linear_regression.ipynb:219]: without removing those $$ marks, pdflatex cannot render the math here.

File: notebooks/source/time_series_forecasting.ipynb [hunk 5]
```diff
@@ -122,28 +122,24 @@
    "source": [
     "The model we are going to use is called **Seasonal, Global Trend**, which when tested on 3003 time series of the [M-3 competition](https://forecasters.org/resources/time-series-data/m3-competition/), has been known to outperform other models originally participating in the competition:\n",
     "\n",
-    "$$\n",
     "\\begin{align}\n",
-    "\\text{exp_val}_{t} &= \\text{level}_{t-1} + \\text{coef_trend} \\times \\text{level}_{t-1}^{\\text{pow_trend}} + \\text{s}_t \\times \\text{level}_{t-1}^{\\text{pow_season}}, \\\\\n",
-    "\\sigma_{t} &= \\sigma \\times \\text{exp_val}_{t}^{\\text{powx}} + \\text{offset}, \\\\\n",
-    "y_{t} &\\sim \\text{StudentT}(\\nu, \\text{exp_val}_{t}, \\sigma_{t})\n",
+    "\\text{exp-val}_{t} &= \\text{level}_{t-1} + \\text{coef-trend} \\times \\text{level}_{t-1}^{\\text{pow-trend}} + \\text{s}_t \\times \\text{level}_{t-1}^{\\text{pow-season}}, \\\\\n",
```
Comment (fehiepsi) [inline notebooks/source/time_series_forecasting.ipynb:126]: Using underscore for texts here will give some errors in pdflatex.

File: docs/source/_templates/breadcrumbs.html [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    {% if hasdoc(pagename) and display_vcs_links %}
+      {% if display_github %}
+        {% if check_meta and 'github_url' in meta %}
+          <!-- User defined GitHub URL -->
+          <a href="{{ meta['github_url'] }}" class="fa fa-github"> {{ _('Edit on GitHub') }}</a>
+        {% else %}
+          {% if 'examples/index' in pagename %}
+            <a href="https://github.com/pyro-ppl/numpyro/blob/master/examples/README.rst" class="fa fa-github"> {{ _('Edit on GitHub') }}</a>
+          {% elif 'examples/' in pagename %}
+            <a href="https://github.com/pyro-ppl/numpyro/blob/master/{{ pagename }}.py" class="fa fa-github"> {{ _('Edit on GitHub') }}</a>
```
Comment (fehiepsi) [inline docs/source/_templates/breadcrumbs.html:16]: I added the link to the example here. Codes below this are the default in sphinx-rtd-theme.

File: docs/source/conf.py [hunk 7]
```diff
@@ -65,6 +65,7 @@
     'nbsphinx',
     'sphinx.ext.autodoc',
     'sphinx.ext.doctest',
+    'sphinx.ext.imgconverter',
```
Comment (fehiepsi) [inline docs/source/conf.py:68]: This is needed to render some svg images in tutorials. This extension converts svg images to a format that pdflatex can work with.

**Comment by neerajprad** [comment]
@fehiepsi - I have not reviewed the changes in detail, but if everything is building fine, it looks good to me. We can iterate further if things break.

**Review by neerajprad — APPROVED** [review]

**Comment by fehiepsi** [comment]
It seems that docs is passing now. But CI is failing...

**Pull request merged**